### DISCUSSION TITLE
Create hard link named after single selected file

### DISCUSSION THREAD

**Discussion by agausto** [jarun/nnn#1754] [Q&A] [answered]
I seem to remember nnn previously using the batch mode logic for single selections of files for hard linking so that I could choose '@' and have a hard link created in the destination folder which had the same filename as the selected file.  Now this only appears to work after you select multiple files.  Is this no longer possible in version 4.9 or was I mistaken this whole time?  Thanks

**Comment by N-R-K** [comment, marked as answer]
> Is this no longer possible in version 4.9 or was I mistaken this whole time?

That's correct. It doesn't do the `@` logic for single files anymore.

But since commit 693ba757c0ed2a644aa it will prefill the input box with the selection if there's only one selection and the `-u` flag is active.
